code/modules/surgery/organs/organ_internal.dm

```diff
@@ -132,3 +144,11 @@
 		if(!getorganslot(ORGAN_SLOT_EARS))
 			var/obj/item/organ/ears/ears = new()
 			ears.Insert(src)
+
+/mob/living/carbon/proc/applyOrganDamage(var/d, var/obj/item/organ/organ)
```

Comment:
Since you have to get the organ to apply damage anyways, why is this a carbon proc?

Reply:
tbf I prefer this because we're more than likely not adding organ to simplemobs and people will just spam this proc and be lazy about using proper typing if we put it on /living or on /mob

Reply:
example: all of the vars in mob defines

Reply:
also personally I think this proc should be seperated into damageorgan and healorgan

Reply:
What I was trying to say is that this proc could just be on the organ itself. Or the parameter could be the organ zone instead of the organ.

Reply:
ah you're completely right.

Reply:
I'll make this change in my next commit. I tried to mimic the liver damage system for the all organs, but it uses living/carbon/proc since I imagine it was meant to mimic most other damage systems (brute/toxins/burns/suffocation), but it would be a lot more flexible if tied to the organ.

Reply:
You can do both, make the carbon proc call the organ proc.